.github/workflows/ci-test-ruby.yaml

```diff
@@ -0,0 +1,76 @@
+name: Test Ruby
+
+on:
+  workflow_call:
+    inputs:
+      extraSystemDependencies:
+        description: 'Install additional system dependencies'
+        required: false
+        default: ''
+        type: string
+
+jobs:
+  test-ruby:
+    name: Test Ruby
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:8.0
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: >-
```

Comment:
very optional: might be worth tuning MySQL to use less ram [like we did in the old e2e tests](https://github.com/alphagov/publishing-e2e-tests/blob/main/docker-compose.yml#L67), which should result in slightly faster tests

Reply:
Annoyingly there doesn't seem to be a way to pass in command args for service containers. 😞

Reply:
Don't worry about it for this PR, but are you sure? It looks like the contents of `options` are [passed as the command line to `docker run`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservicesservice_idoptions), so I'd expect to just be able to override the image's `CMD` by adding commands at the end of `options` (like `"mysqld", "--foo", "--bar"). I haven't actually tried this though, so apologies if this doesn't work.